.circleci/config.yml

```diff
@@ -42,6 +42,10 @@ jobs:
         name: 'Run JVM test suite'
         command: lein with-profile -dev test
 
+    - run:
+        name: 'Run JVM test suite, failing on reflection warnings'
+        command: lein with-profile -dev,+warn-on-reflection test 2>&1 | grep warning && exit 1 || exit 0
```

Comment:
We could try to capture the output of the first test-run (L43), and only check the output in a similar way. That prevents the whole test-suite to run.

Reply:
Yes, definitely. Such a setup could be even reused in all our projects.

Not immediately obvious how to achieve that though...

Two requirements:

* keep 'streaming' the output to Circle
* make it clear if the build failed due to red tests or due to grep

We can eventually give it a shot.